net-fs/nfs-utils/nfs-utils-2.6.3.ebuild

```diff
@@ -126,6 +126,7 @@ src_configure() {
 		$(use_enable nfsv41)
 		$(use_enable uuid)
 		$(use_with tcpd tcp-wrappers)
+		LIBS="-lsqlite3 -levent_core"
```

Comment:
Please fix the configure script and send the fix upstream.

Reply:
I sent fix upstream: https://git.linux-nfs.org/?p=steved/nfs-utils.git;a=commit;h=bc4a5deef9f820c55fdac3c0070364c17cd91cca

But for nfs-utils-2.6.3 these libs shoud be passed in ebuild.